packages/apps/lightning-ui-docs/.storybook/addons/decorators/withLightning.js

```diff
@@ -130,6 +130,11 @@ export const withLightning = (
             this._refocus(); // Force Lightning to reset focus
           }
           _setup() {
+            // This ensures the component as it's args before the first update cycle. Fixes some flashing in storybook.
+            // TODO: confirm if this can be the whole args object or if we have to process it somehow
+            this.componentTarget.patch({
+              ...args
```

Comment:
Is there anything we have to do to args here? It appears to just contain the props coming from storybook, but I see there is some conditional handling of it below...